geps/gep-713.md

```diff
@@ -649,6 +1135,29 @@ const (
 )
 ```
 
+#### On targeted resources
+
+Implementations that use Policy objects MUST ensure that a Condition exists in
+the `status.Conditions` of any objects affected by a Policy. If that Condition is
+already set to `status: true`, then the implementation must ensure that
+the `observedGeneration` field matches the `metadata.generation` field of the
+object.
+
+Additionally, if an implemntation sees an update that changes the `status`` of this
+Condition from `false` to `true` _that it did not create_, it must not attempt
+to change the status back to `false`.
```

Comment:
Deploy preview struggles here, I think it will be fixed by this change, but worth double checking: https://deploy-preview-2128--kubernetes-sigs-gateway-api.netlify.app/geps/gep-713/#on-targeted-resources

```suggestion
Additionally, if an implementation sees an update that changes the `status` of this
Condition from `false` to `true` _that it did not create_, it must not attempt
to change the status back to `false`.
```